f1_racing_results.py

```diff
@@ -0,0 +1,4 @@
+from formula1_race_analysis import generate_report
+
+if __name__ == "__main__":
+    generate_report()
```

Comment:
Also, please create one big def cli(): and inherit all the commands from it. Generate report might be not the only command you get